### ISSUE TITLE
CYS: investigate if patterns and fonts can be installed in parallel

### ISSUE THREAD

**Issue by Aljullu** [woothemes/woocommerce#51983]
As discussed in https://github.com/woocommerce/woocommerce/pull/51819#discussion_r1784279981, there might be an optimization we could do to the Customize Your Store flow when tracking is enabled, and we install patterns and fonts: we could try to install them in parallel instead of sequentially. When working on this, we need to make sure it doesn't introduce any regressions.

The relevant code can be found here:

https://github.com/woocommerce/woocommerce/blob/6a5a4f14d1d5451d218329efea7e8cac26344fd4/plugins/woocommerce-admin/client/customize-store/assembler-hub/opt-in/opt-in.tsx#L141-L142

**Comment by mdperez86** [comment]
Moved this issue to `on hold` since CYS project is `on hold` now.

**Comment by tyxla** [comment]
No longer relevant since the CYS flow was removed in [https://github.com/woocommerce/woocommerce/issues/61932](https://github.com/woocommerce/woocommerce/issues/61932). Closing.
